lib/data_pull.rb

```diff
@@ -263,4 +266,48 @@ def run(args:, config:)
       )
     end
   end
+
+  class ProfileSummary
+    def run(args:, config:)
+      uuids = args
+
+      users = User.includes(:profiles).where(uuid: uuids).order(:uuid)
+
+      table = []
+      table << %w[uuid current_status activated_timestamp disabled_reason
+                  gpo_verification_pending_timestamp fraud_review_pending_timestamp
+                  fraud_rejection_timestamp]
+
+      users.each do |user|
+        if user.profiles.any?
+          user.profiles.sort_by(&:id).each do |profile|
+            profile_status = "profile_id: #{profile.id} (#{profile.active ? 'active' : 'inactive'})"
```

Comment:
what if we make profile id its own column, so we don't have to waste horizontal space saying `profile_id` inside the status col?

Reply:
![image](https://github.com/18F/identity-idp/assets/109746710/3774aff1-b7bc-497d-b298-5148a67d287f)